netmap/policy_internal_test.go

```diff
@@ -0,0 +1,99 @@
+package netmap
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/nspcc-dev/neofs-api-go/v2/netmap"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPlacementPolicy_CopyTo(t *testing.T) {
```

Comment:
can we have simple test (here and everywhere)?:
```go
src := netmaptest.PlacementPolicy()
var dst PlacementPolicy

src.CopyTo(&dst)
require.Equal(t, src, dst)

var empty netmap.PlacementPolicy
empty.CopyTo(&dst)

require.Equal(t, empty, dst)
```

Reply:
this may be a small sub-test. I don't see any `require.Equal` comparisons that assert deep equality

Reply:
I'm comparing marshaled results, but added `require.Equal` as second check

Reply:
Added `require.Equal` to all tests